.github/workflows/pull-requests.yaml

```diff
@@ -0,0 +1,70 @@
+name: Build-Deploy dev
+on: [pull_request]
```

Comment:
Vi gikk tidligere bort fra å bygge on pull request fordi det gikk i bein på hverendre, og det kom databas-endringer som ikke skulle bli deployet :P 

Hadde det vært greit å kjøre denne etter deploy-prod? Dvs kjørt smoketest som en del av build-prod, eks i sluttet? Ulempen er att man allerede har deployet når man inser att smoketest feiler.

Reply:
Vil ikke deploye PR-er til preprod, nei. Det blir kaotisk å ha oversikt - og en liten mini-fiks skriver plutselig over en stor branch som skal testes. Èn ting er master, den _skal_ ut, men en PR skal ikke nødvendigvis testes i preprod før QA.

Reply:
Hvis smoketesten ikke er for krevende går det an å legge den inn i `deploy-dev.yaml`-kjøringen som trigges med workflow-dispatch, ellers kan den dundres ut i `deploy-prod.yaml` - så får vi leve med litt "reaktiv" håndtering (som smoketestene i ef-sak-frotend)

Reply:
Jeg synes egentlig ikke at det er så ille, men er litt enig med dere. Men det høres ut som en grei løsning det synes jeg: Legger det inn i deploy-dev som trigges av workflow dispatch, og i deploy-prod.